src/components/Employee/Deductions/stateMachine.ts

```diff
@@ -67,6 +71,16 @@ export const deductionsStateMachine = {
         }),
       ),
     ),
+    transition(
+      componentEvents.EMPLOYEE_DEDUCTION_DELETED_EMPTY,
+      'includeDeductions',
+      reduce(
+        createReducer({
+          component: IncludeDeductionsFormContextual,
```

Comment:
Wondering if we rename this component to say empty state in the name since that appears to be what it's being used for now?